css/style.css

```diff
@@ -545,6 +1071,246 @@ body {
     height: 20px;
     width: 20px;
   }
+
+  .product-card__btn:hover {
+    height: 20px;
+    width: 20px;
+    opacity: .5;
+    cursor: pointer;
+  }
+
+  .video {
+    flex-direction: column-reverse;
+    min-width: 768px;
+    margin-bottom: 0px;
+    background: linear-gradient(2.6deg, #000 95.3%, #fff 95%);
+  }
+
+  .video__header {
+    text-align: center;
+    font-size: 30px;
+    line-height: 37px;
+    margin-bottom: 15px;
+  }
+
+  .video__text-wrapper {
+    width: 665px;
+    height: 134px;
+    margin-top: 88px;
+    margin-left: 10px;
+  }
+
+  .video__text {
+    text-align: center;
+    font-size: 20px;
+    line-height: 25px;
+  }
+
+  .video__link {
+    display: none;
+  }
+
+  .video__frame {
+    background: url(../img/video-bg-768.png) no-repeat;
+    width: 705px;
+    height: 497px;
+  }
+
+  .video__btn {
+    height: 40px;
+    width: 40px;
+    margin-bottom: 106px;
+    margin-right: 12px;
+  }
+
+  .video__btn::before {
+    left: 13px;
+    bottom: 12px;
+  }
+
+  .stories {
+    background: linear-gradient(2.6deg, #fff 20.5%, #000 21%);
+  }
+
+  .stories-card:last-child {
+    display: none;
+  }
+
+  .stories__header {
+    font-size: 30px;
+    line-height: 29px;
+    color: #fff;
+  }
+
+  .stories__card-wrapper {
+    max-width: 724px;
+    justify-content: normal;
+    padding-left: 44px;
+    margin-bottom: 127px;
+  }
+
+  .stories-card:first-child {
+    padding-right: 62px;
+  }
+
+  .stories-card__img_1024 {
+    display: none;
+  }
+
+  .stories-card__img_768 {
+    display: block;
+    margin-bottom: 30px;
+  }
+
+  .stories-card__header {
+    font-size: 25px;
+    line-height: 24px;
+    color: #fff;
+    margin-bottom: 10px;
+  }
+
+  .stories-card__text {
+    font-size: 20px;
+    line-height: 25px;
+    color: #fff;
+    width: 312px;
+    margin-bottom: 34px;
+  }
+
+  .stories-card__link {
+    font-size: 20px;
+    line-height: 19px;
+    color: #fff;
+    width: 200px;
+  }
+
+  .stories-card__link::after {
+    left: 153px;
+    bottom: 0px;
+    background: url(../icons/arrow-768.png);
+    width: 49px;
+    height: 15px;
+  }
+
+  .stories-card__date {
+    font-size: 15px;
+    line-height: 14px;
+    color: #d1d1d1;
+    opacity: 1;
+  }
+
+  .stories-card__footer {
+    width: 332px;
+  }
+
+  .stories-card:nth-child(2) .stories-card__footer .stories-card__date {
+    margin-right: 21px;
+  }
+
+  .stories__subheader {
+    font-size: 30px;
+    line-height: 29px;
+    color: #E8E8E8;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#E8E8E8" to be "#e8e8e8" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5612618)